From db143115fa178a5b5ee1d7c490584c9d833dd431 Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Tue, 19 Feb 2019 17:54:52 -0700
Subject: [PATCH 04/11] Revert "legacy: allow init to read /proc/device-tree"

* We shouldn't be labeling a proc node as a sysfs_type. We'll go
  ahead and handle this properly in system/sepolicy.

This reverts commit 25b6ab3f932d821281893123a37f6d6c3621b17b.

Change-Id: I38b7e237a438b779be37a7f071d8d6379e2bdb7c
---
 legacy-common/genfs_contexts | 2 --
 1 file changed, 2 deletions(-)

diff --git a/legacy-common/genfs_contexts b/legacy-common/genfs_contexts
index 4781836..f94e9eb 100644
--- a/legacy-common/genfs_contexts
+++ b/legacy-common/genfs_contexts
@@ -1,3 +1 @@
 genfscon debugfs /rmt_storage u:object_r:debugfs_rmt:s0
-
-genfscon proc /device-tree/firmware/android u:object_r:sysfs_dt_firmware_android:s0
-- 
2.17.1

